cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,46 @@
-describe('', () => {
-  before(() => {
-
+describe('Conduit article', () => {
+  let article = {};
+  let user = {};
+  beforeEach(() => {
+    cy.visit('https://conduit.mate.academy/');
+    cy.task('generateUser').then((newUser) => {
+      user = newUser;
+    });
+    cy.task('generateArticle').then((newArticle) => {
+      article = newArticle;
+    });
   });
 
-  it('', () => {
-
+  it('should be able to create a new article', () => {
+    cy.signUp(user.email, user.username, user.password);
```

Comment:
SignUp can be added to beforeEach if you want. 
In the first test, we need to add an article manually and use a function only in the second one. 
also
Try not to repeat much code between tests.

Reply:
Done. But putting signup in before each wasn't easy ; d